app/src/org/commcare/android/database/user/models/FormRecord.java

```diff
@@ -69,6 +69,12 @@
     @Persisting(6)
     @MetaField(META_LAST_MODIFIED)
     private Date lastModified;
+
+    /**
+     * Unique id of the CC app to which this form belongs
+     */
+    @Persisting(7)

```

Comment:
Also: If we changed this model we'll need to do a migration on its database or unsaved/unsent forms will become corrupted after upgrade.